Config/Src/config_msgs.c

```diff
@@ -0,0 +1,58 @@
+/**
+ * This is a C file because we will include it in compilation,
+ * so all these messages are only executed once.
+ * 
+ * The general format is as follows:
+ * 
+ * #ifdef COMPILER_DEFINE_HERE
+ * #pragma message ("XYZ WAS SPECIFIED BY COMPILER")
+ * #endif
+ * 
+ * config.h will also have similar formatting.
+ */
+
+#ifdef CUSTOM_COMMAND_SIZE
+#pragma message ("Custom command size set!")
+#endif
```

Comment:
I think we want to print out the values of the macros as well (assuming it's not obvious from looking at the make output). The motivation for this is that there were a few times last year we spent time debugging "bugs" when we had the wrong configuration in config.h

Reply:
I'll try, unsure if the preprocessor supports this explicitly but there's probably a way to do it